### PR TITLE
[WIP] Low cooking skill fatty food for the new hunger system.

### PR DESCRIPTION
**Pull request by Lovewyrm**
#### Summary
SUMMARY: Content "Add more calory dense food for low cooking skill/primitive characters"

#### Purpose of change
The hunger system tries to emulate stomach fullness, making high calory dense food more attractive for building calory storage or just to offset manual labor.
The early game lacks fatty foods that don't completely slash mood. There is tallow and lard, but those are quite the mood killers, while also being more difficult to produce irl than just roasting a slab of pork belly and co.
Raw fat has great caloric value, but of course, uncooked is full of potential parasites and co.

Adding low/unskilled filling, calory dense meals for carnivores that are overall neutral mood wise, will bridge the gap between cooking levels of higher tier items, which often give greater mood bonuses, and co.

There isn't a good way to emulate utterly basic calory rich diets for unskilled/primitives even though the realworld does so easily thanks to fatty tissue in (prey) animals.
The purpose of this set of (if approved) growing recipes is to free the player from the scourge of eating cat tails, tallows and lard in the early game.
Freeing the player to snack on wild herbs and an egg or two for breakfast on the regular, but giving them also a way to bulk up without ruining their day.
Fat is delicious, it should give mood bonuses, but I am not adding any to these recipes because they are unseasoned and low skill, this is purely for calories versus 'stomach fillage'.

#### Describe the solution
In this draft, I chose 1 chunk of fat for 4 chunks of meat to create 4 portions worth of 880 calories of bland, but nourishing cutlets.
I chose this unbalanced recipe because a chunk of fat is 2000 calories, and I figured that it's easier to just demand a bulk of meat to 'cut the fat' with, than creating yet another item to manage, like 'strip of fat' which could then perhaps go 1:1 with meat.
(In other words, 1 chunk of fat of 2k calories, into 4 strips of fat, then 1strip + 1chunk of meat = 1 fatty cutlet)

Also, cracklins are unhealthy, but I do acknowledge their existence, but in game I usually just feed my characters lard and tallow, not fun at all.

#### Describe alternatives you've considered
The only alternatives I can think of is tweaking numbers. Even the dumbest ape can just sink his face into a carcass and chow on the blubber, it's not healthy of course, but it also doesn't take a genius to just hold some marbled belly meat over a fire, and since the game treats fat and meat as separate things, this is the only approach I can see without major changes.

Also, chunks of fat could be turned into strips to simplify recipes, but that would add another item type, too.

#### Testing
Standard ingame testing, recipe works, but numbers can probably be tweaked, maybe a small caloric loss, maybe make it healthy -1, etc.

#### Additional context

This is just a draft, if the concept is approved/tweaked I will add more recipes.

P.S.:
I am still new both at the game and therefore also the contribution process.
If 'drafts' like these are bad form on PRs, please tell me proper conduct to fix that. IRC channels, discord, whatever.

### CODE REVIEW

**Review by venum4k — CHANGES_REQUESTED** [review]
Just to keep it in-line with another ongoing PR on food recipes, I think it would make sense to have the activity level as LIGHT_EXERCISE.

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -151,6 +151,17 @@
     "tools": [ [ [ "surface_heat", 1, "LIST" ] ] ],
     "components": [ [ [ "meat_scrap", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
```
Comment (venum4k) [inline data/json/recipes/recipe_food.json:156]:
```suggestion
    "activity_level": "LIGHT_EXERCISE",
```

**Review by venum4k — CHANGES_REQUESTED** [review]
First time doing this request changes thing. there we go...

File: data/json/recipes/recipe_food.json (hunk 1 above)
Comment (Lovewyrm) [reply data/json/recipes/recipe_food.json]: I have added this to the new commit.

**Comment by anothersimulacrum** [comment]
Drafts are more than fine! They're probably better than just keeping it silent, then you can get review early and maybe prevent two people working on it at the same time.

**Comment by Lovewyrm** [comment]
Made the fatty cuts more generalized, like dried meat, to make the game handle the calculations of calories instead of the human.

Numbers still need tweaking, 4 cuts of meat for 1 chunk of fat is a bit iffy. 
220g of fat for 1200g of meat (1 chunk of fat + 4 meat) is quite the fatty meal, beyond greasy burger, but I don't think that's really a problem in a primitive setting since even though lean meat outweighs fatty meat, the fat can still be used to 'fortify' the meat by adding it as an ingredient.
(As in, if you slaughtered a cow IRL and you'd use every single cut of fat to add them to any cut of meat, lean or otherwise, you'd still have a 1:1 consumption ratio of fat vs meat, aka, the entire meat of the cow together with the entire fat of it)


It's just that, after looking through harvest.json, it seems that most mammals/animals have ~4 times more meat than fat, which would allow for making fatty cuts from the entire animal.

This looks unbalanced, but using all of the fat also removes it  from any other recipes, and could just be the above scenary, where all fat of the animal is used by putting it in the roasts/food. So it's technically okay, but from a gameplay perspective it might 'feel' (don't like putting it that way, but, lack of words) like the only way to go, or something.

But that could just be the games inherent awkwardness considering fat and meat in the new nutrition system.
Fish, for example, some fish are naturally fatty, but there are no fatty cuts from the fish to be had.

So now I'm wondering if there shouldn't be a fatty and lean cut ingredient that's harvestable, and then unify those cuts under the respective 'meat umbrella' of the species, and have the game solve it like dry meat, where whichever dish you make from the cuts, either lean or fatty, increases or decreases the final caloric value.

So, a sandwich with a lean cut would have less calories than one with a fatty cut, but both fatty and lean are unified as a component for making the recipe, like how 'edible_lard' is various lards, etc.

It would solve this awkwardness, but also not really, because, again, theoretically you could use every single scrap of fat on any animal to just throw it into the roast/cutlet/dish.

What is the point of me writing all this then?
I am not too sure, I guess just to write it out, to bounce off of. Both ways work, but I'm concerned about how a player would perceive this situation.

It works as it is, right now, survivors can't be choosers, and I figure that anyone in a life or death situation who just butchered an animal wouldn't just try to fry it all up, so 4 (or any other amount of ) meat + 1 chunk of fat works.

Again, this is just musing. WIP, and all that.

**Comment by I-am-Erk** [comment]
Is this stalled? If you want we can get this basic stuff in game rather than leaving this as a WIP forever. Nothing wrong with multiple small PRs

**Comment by Lovewyrm** [comment]
> Is this stalled? If you want we can get this basic stuff in game rather than leaving this as a WIP forever. Nothing wrong with multiple small PRs

It's stalled, or rather, I don't want to contribute to CDDA anymore, doesn't matter how little I did so far.
In fact, I want what I have contributed, even though it was just that check for firewood, removed from the game.

The reason for that is that I don't want to be associated with horseshit like "now with less racism" to rename some shotgun because some author or something of some recipe out there is deemed racist.

I've seen my share of this kind of poisoning and I don't even want an internet handle of mine associated with that.

**Comment by I-am-Erk** [comment]
Well, I find it hilarious that your reason for leaving is fragility over a commit comment joke by a single contributor, but that's fine. It's a volunteer project.

**Comment by Lovewyrm** [comment]
>fragile
You mean like unable to handle the name "four winds shotgun"?

>a commit comment joke

image

My reason is not fragility, it's that I don't want to be associated with awful people. That's all.

**Comment by I-am-Erk** [comment]
The name "four winds shotgun" is a stupid name: it doesn't describe the weapon, it is not the common term for a slamfire shotgun, and therefore not appropriate for the project. The original author acknowledged that they only used it because it sounded cool, and that it wasn't a good choice for readability. That is the primary reason to remove it. The fact that it was originally promoted by a neo-nazi is just icing on the cake. That's clearly explained in the post you're quoting, right in front of you. The only people getting into a froth are people who see a negative comment about racism as a dog whistle and get defensive about it, as though acknowledging that racism is a thing is somehow an attack on you. The rest of us are quite content to change it to a widely recognizable term for the weapon and get on with our lives.

**Comment by Lovewyrm** [comment]
I'm not a horses ass, I just don't want to be associated with awful people.

And yes, I do consider such a thing a 'dogwhistle', and you promptly fulfilled it with textbook diatribe, including 'fragility' and 'dogwhistle'.

I am also not in a froth, I have just seen enough of this kind of crap and I don't want to be associated with it over something as irrelevant as a game.
And no, you're not content to just move on with your lives.

Someone with integrity like that would have answered:
Fair enough, we'll remove your contributions.

Not the song of dance of fragility and co.

**Comment by ZhilkinSerg** [comment]
> I want what I have contributed, even though it was just that check for firewood, removed from the game.

What are you talking about?

**Comment by esotericist** [comment]
> > I want what I have contributed, even though it was just that check for firewood, removed from the game.
> 
> What are you talking about?

presumably #43258

although reverting that would be kinda ridiculous, and unwarranted.

**Comment by I-am-Erk** [comment]
Please see the readme section under license, or more specifically:  https://github.com/CleverRaven/Cataclysm-DDA/blob/master/.github/CONTRIBUTING.md

> Cataclysm:Dark Days Ahead is released under the Creative Commons Attribution ShareAlike 3.0 license. The code and content of the game is free to use, modify, and redistribute for any purpose whatsoever. See http://creativecommons.org/licenses/by-sa/3.0/ for details. This means any contribution you make to the project will also be covered by the same license, and this license is irrevocable.

Structurally, it's a thing we just don't do for a multitude of reasons.

**Pull request closed without merging**